src/components/PreviewList.js

```diff
@@ -114,6 +109,7 @@ function PreviewList({
 PreviewList.propTypes = {
     classes: PropTypes.object.isRequired,
     fileObjects: PropTypes.arrayOf(PropTypes.object).isRequired,
+    getPreviewIcon: PropTypes.func,
```

Comment:
Mark `getPreviewIcon` as `isRequired`.

Reply:
Oh yes I see, it is always being passed in anyway but this will enforce it as mandatory, not having it will have potentially undesired effects